packages/api/src/external/aws/appConfig.ts

```diff
@@ -0,0 +1,28 @@
+import { getFeatureFlagValue } from "@metriport/core/external/aws/appConfig";
+import { Config } from "../../shared/config";
+import { errorToString } from "../../shared/log";
+import { capture } from "../../shared/notifications";
+
+export async function getCxsWithEnhancedCoverageFeatureFlagValue(): Promise<string[]> {
+  try {
+    const cxIds = await getFeatureFlagValue<string[]>(
+      Config.getAWSRegion(),
+      Config.getAppConfigAppId(),
+      Config.getAppConfigConfigId(),
+      Config.getEnvType(),
+      Config.getCxsWithEnhancedCoverageFeatureFlagName()
```

Comment:
NABD, but do we need both the region and the env type?

Reply:
The configuration in AppConfig is named after the environment, it just so happens that we have our environments deployed in different regions - but we don't exactly have env/region mapping anywhere, and since this could change, probably not worth hardcoding